google/cloud/bigquery/magics/magics.py

```diff
@@ -89,15 +89,16 @@
 import sys
 import time
 import warnings
+import threading
 from concurrent import futures
 
 try:
     import IPython  # type: ignore
     from IPython import display  # type: ignore
     from IPython.core import magic_arguments  # type: ignore
 except ImportError:  # pragma: NO COVER
-    raise ImportError("This module can only be loaded in IPython.")
-
+    raise  # ImportError("This module can only be loaded in IPython.")
+import ipywidgets as widgets
```

Comment:
Should this be in the try block as well?  Put another way, are there cases where importing ipywidgets has merit without ipython presence?

Reply:
Very true - I'm wondering if I should change the exception flag at all, though.